### DISCUSSION TITLE
About the Name Cozy

### DISCUSSION THREAD

**Discussion by ayo-run** [ayo-run/Cozy#72] [General]
I cleaned up the project README and decided to keep the "Cozy" name there, but I want to open this discussion to keep the idea of renaming the project open.

In this [short post](https://social.ayco.io/@ayo/110547172702258070), I mentioned another reader app with the same name (Cozy Reader) but has significant differences in this project's approach.

Several suggestions were given to me and I will link in here later, but I also still like to consider keeping "Cozy" -- just Cozy, so we don't say "Cozy Reader" anymore.

**Comment by exstntlmsnthrp** [comment]
I appreciate the transparency and appreciate your point. 👍

**Comment by ayo-run** [comment]
Update on the name
---

I wanted to link all suggestions made, but my instance of Mastodon's lack of text search is making it hard! 😅

Anyway, the most prominent suggestions was related to the term [koselig](https://www.thatscandinavianfeeling.com/lifestyle/norwegian-concept-koselig) and variations of it. I think it captures the general feel this project aims to have.

However having a short domain name is of practical importance since we want to make it easy to open and share articles in the app.

### New name: Ko-zi

So after research of available domain names, I think "Ko-zi" is a good short name.

There will probably be shorter names and other available domains but I feel good about this one. When I raise enough funds I'll get a domain for the app.
